### ISSUE TITLE
InputDate with Type=InputDateType.Time and step=1 doesn't accept keyboard input

### ISSUE THREAD

**Issue by SteveSandersonMS** [dotnet/aspnetcore#38471]
Using `<InputDate Type="InputDateType.Time" @bind-Value="someDateTimeField" step="1" />`, you'll be unable to type normally into the input. After each keystroke it behaves as if you've just moved the cursor back to the start, and your next keystroke overwrites the last. This manifests as a test failure in our `InputDateInteractsWithEditContext_TimeInput_Step` E2E test which covers this exact scenario. Or you can repro it manually:

 * Run `BasicTestApp` and go to *Typical Validation* using Edge/Chrome
 * Next to *Departure time*, check the *Include seconds* box
 * Click inside the textbox below it and start typing 123456.
 * Expected: time shows as `12:34:56`
 * Actual: depending on how quickly you type, you'll get results like `01:00:00` or `02:00:00` or `03:00:00` with subsequent keystrokes ignored

This does seem to be a browser bug, because:

 * It doesn't happen on Firefox. Typing is totally normal there
 * It doesn't happen with `datetime-local`. It's very specific to `type=time` with `step=1` when the previous value ends with `:00`.

Even if it is a browser bug, it's possible that Blazor might be able to work around it. It's hard to repro this issue without involving Blazor so it's probably specific to the way our data binding interacts with the exact sequence of events.

**Comment by mkArtakMSFT** [comment]
Hi. Thanks for contacting us.
We're closing this issue as there was not much community interest in this ask for quite a while now.
You can learn more about our triage process and how we handle issues by reading our [Triage Process](https://github.com/dotnet/aspnetcore/blob/master/docs/TriageProcess.md) writeup.
